lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp

```diff
@@ -2861,6 +2861,62 @@ bool FoldChannelShuffle::run(Function *F, const CompilationContext &cctx) {
   return changed;
 }
 
+// Fold Tile -> Add into BatchedAdd wherever applicable.
+bool FoldTileAddIntoBatchedAdd::run(Function *F,
+                                    const CompilationContext &cctx) {
+  LOG_SCOPE(F->getLogContext(), getName());
+
+  bool changed = false;
+  auto &nodes = F->getNodes();
```

Comment:
It doesn't look like this is used anywhere else so I think you can plug the definition directly into the for loop.

Reply:
Done!